docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
 
-## 2. Features
+## 3. Features
 
 > **Command Format**
 >
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
+> * Items surrounding `|` means we only choose one of them (OR).
 > * Items with `...` after them can have multiple instances.
-> * Parameters can be in any order.
+> * Parameters should be in the order stated (at least for now).
 
-### 2.1. Viewing help : `help`
+### 3.1. Viewing help : `help`
 
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 3.2. Adding an item: `adde` or `addt`
+
+Format (event):
+
+`adde "event name" START_DATE START_TIME to [END_DATE] END_TIME [t/TAG]... [//DESCRIPTION]`<br>
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Format (tentative event. blocks multiple time slots): 
 
-> Persons can have any number of tags (including 0)
+`adde "event name" START_DATE START_TIME to [END_DATE] END_TIME or ... [t/TAG]... [//DESCRIPTION]`<br>
+
+Format (task with deadline): 
+
+`addt "task name" DEADLINE [t/TAG] [p/PRIORITY] [//DESCRIPTION]`  <br>
+
+Format (task without deadline): 
+
+`addt "task name" [t/TAG] [p/PRIORITY] [//DESCRIPTION] `<br>
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `adde "Cs2103 meeting" 3 Mar 17:00 to 18:00`
+* `addt "Buy tickets" 3 mar 17:00`
+* `addt "Take GF for dinner"`
+
+### 3.3 Viewing the active/expired list: `list`
 
-### 2.3. Listing all persons : `list`
+**View the entire active list**: 
 
-Shows a list of all persons in the address book.<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
+**View tasks with/without deadline**: 
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format: `listd [DEADLINE] [SORT_FIELD]`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
-    The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
-> * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+Example:
 
-Examples:
+* `listd 3 March`
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+**View events within a time frame, or with no time frame specified** : 
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+Format: `liste [START_DATE START_TIME to END_DATE END TIME] [SORT_FIELD]`
```

Comment:
same comment as `adde/addt` for all list variants. 
think how you can indicate what should be done without having to provide so many commands, can?